scripts/s3-sanity/s3-sanity-test.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
if(s3imacli)
{
    create acc
    create user
      if(s3cmd)
      {
      create bucket
      create object
      delete bucket
      delete object
      }
    delete accd
    delete user  
}

Reply:
The previous code was following this pattern. Please review the current code which is slightly restructured.